Source/Classes/NantesLabel.swift

```diff
@@ -487,17 +500,17 @@ public extension NSAttributedString.Key {
         return textRect
     }
 
-    /// We're handling link touches elsewhere, so we want to do nothing if we end up on a link
+    /// If there's a link at the touch point, handle that in touchesEnded
+    /// If there's a label tapped block, don't forward touches, run the block
+    /// Otherwise forward touches on
     override open func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
-        guard let touch = touches.first,
-            let activeLink = link(at: touch.location(in: self)) else {
-                if labelTappedBlock == nil {
-                    super.touchesBegan(touches, with: event)
-                }
-                return
+        if let touch = touches.first, let activeLink = link(at: touch.location(in: self)) {
+            self.activeLink = activeLink
+        } else if let labelTappedBlock = labelTappedBlock {
```

Comment:
You probably don't want to do this as it causes the labelTappedBlock to be called twice for each tap (so using the example of toggling the number of lines, it causes it to expand on touchesBegan but then contract again on touchesEnded)

Reply:
👍 good catch, thank you